flow/cmd/mirror_status.go

```diff
@@ -190,10 +198,43 @@ func (h *FlowRequestHandler) cdcFlowStatus(
 		SnapshotStatus: &protos.SnapshotStatus{
 			Clones: initialLoadResponse.TableSummaries,
 		},
-		CdcBatches: cdcBatchesResponse.CdcBatches,
+		CdcBatches: cdcBatches,
+		RowsSynced: rowsSynced,
 	}, nil
 }
 
+func (h *FlowRequestHandler) CDCGraph(ctx context.Context, req *protos.GraphRequest) (*protos.GraphResponse, error) {
+	truncField := "minute"
+	switch req.AggregateType {
+	case "1hour":
```

Comment:
could this be made a proto enum?

Reply:
I'd rather not, it's postgres interval syntax, don't trust protobuf to preserve strings easily